src/Streamly/Internal/Data/Stream/IsStream/Exception.hs

```diff
@@ -167,7 +168,24 @@ bracket_ bef aft bet = D.fromStreamD $
 bracket :: (IsStream t, MonadAsync m, MonadCatch m)
     => m b -> (b -> m c) -> (b -> t m a) -> t m a
 bracket bef aft bet = D.fromStreamD $
-    D.bracket bef aft (toStreamD . bet)
+    D.bracket bef aft aft aft (toStreamD . bet)
```

Comment:
I don't know if I like the inconsistency between the API between StreamD and IsStream. We could add another combinator and eventually replace bracket with it.

Reply:
We can call it `D.bracket'`, it will be consistent with `bracket'`.